src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -294,17 +295,15 @@ export class SavedObjectsService
 
     this.setupDeps = setupDeps;
 
+    this.typeRegistry.registerType(configSavedObjectType);
+
```

Comment:
I'm registering the `config` type from the so service itself. @restrry do you think it make more sense to have the type declaration in `src/core/server/ui_settings` and having the uiSettings service performs the registration instead?

Reply:
Can do in this PR, moving the type to the `ui_settings` folder should be trivial